tune-threads/utilities.lisp

```diff
@@ -0,0 +1,49 @@
+;;;; tune-threads/tune-threads.lisp
+;;;;
+;;;; Author: John Lapeyre
+
+(in-package #:qvm-tune-threads)
+
+(defun mean-dev-from-sums (x-sum x-sq-sum n)
+  "Given a sum X-SUM and sum of squares X-SQ-SUM of N iid random samples, compute the mean and the standard deviation"
+  (let ((x-mean (/ x-sum n)))
+    (list x-mean (sqrt (- (/ x-sq-sum n) (expt x-mean 2))))))
+
+(defun findmin (list)
+  "Return a list whose two elements are the minimum value of LIST and the index of that value.
+If the minimum is degenerate, the fist index is returned."
+  (do* ((min-val (car list))
```

Comment:
Also, I don't want to put words in anyone's mouth, but my guess is @stylewarning won't be too pleased with the choice of `do*` here :). Prefer `loop` where possible.

Reply:
I've got no problem with that. But, it would be a good idea to redesign the code that calls `findmin` at a higher level in any case.

Reply:
Replaced `do` with `loop`.